### PR TITLE
pyats-version: add page

### PR DESCRIPTION
**Pull request by barweiss45**
- [*] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [*] The page(s) have at most 8 examples.
- [*] The page description(s) have links to documentation or a homepage.
- [*] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [*] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/pyats-version.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Display out-dated packages:
+
+`pyats version check --outdated`
+
+- Update to most recent version:
+
+`pyats version update`
+
+- Update or downgrade to to specific version:
```
Comment (kbdharun) [inline pages/common/pyats-version.md:18]:
```suggestion
- Update or downgrade to a specific version:
```

File: pages/common/pyats-version.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Display version of all packages:
+
+`pyats version check`
+
+- Display out-dated packages:
+
+`pyats version check --outdated`
+
+- Update to most recent version:
```
Comment (kbdharun) [inline pages/common/pyats-version.md:14]:
```suggestion
- Update to the most recent version:
```

File: pages/common/pyats-version.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# pyats version
+
+> To view and upgrade the pyATS installation.
+> More information: <https://developer.cisco.com/pyats/>.
+
+- Display version of all packages:
+
+`pyats version check`
+
+- Display out-dated packages:
```
Comment (kbdharun) [inline pages/common/pyats-version.md:10]:
```suggestion
- Display outdated packages:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**